webui/react/src/components/LogViewer/LogViewer.stories.tsx

```diff
@@ -23,7 +22,7 @@ const messageWithTags = `continuing trial: <COMPUTE_VALIDATION_METRICS: (10,10,3
 export const Default = (): React.ReactNode => (
   <div style={{ height: '100vh' }}>
     <LogViewer
-      decoder={jsonToClusterLog}
+      decoder={mapV1LogsResponse}
```

Comment:
what motivated this?

Reply:
LogView doesn't display with `jsonToClusterLog`, so I changed it to `mapV1LogsResponse`